pkg/solana/logpoller/log_data_parser.go

```diff
@@ -78,8 +80,9 @@ func parseProgramLogs(logs []string) []ProgramOutput {
 
 			if len(dataMatches) > 1 {
 				instLogs[lastLogIdx].Events = append(instLogs[lastLogIdx].Events, ProgramEvent{
-					Prefix: prefixBuilder(depth),
-					Data:   dataMatches[1],
+					Program: instLogs[lastLogIdx].Program,
+					Prefix:  prefixBuilder(depth),
```

Comment:
If `Prefix` isn't needed/used, we can get rid of it. The idea is that it shows overall log nesting.

Reply:
Good point, as far as I know we don't need it